tests/utilities.py

```diff
@@ -0,0 +1,9 @@
+from yarl import URL
+
+
+def latest_request(mocked, request_type, path):
```

Comment:
Should we reuse these helpers for the core / nlu tests and remove them from the core / nlu test utilities?